packages/documentation/copy/en/javascript/JSDoc Reference.md

```diff
@@ -362,9 +383,19 @@ function seriousalize(key, object) {
 }
 ```
 
-Declaring generic classes or types is unsupported.
+Finally, you can specify a default for a type parameter:
+
+```js twoslash
+/** @template [T=object] */
+class Cache {
+    /** @param {T} initial */
+    constructor(T) {
+    }
+}
+let c = new Cache()
+```
 
-## Classes
+# Classes
```

Comment:
We might need to bump all of them up a `#` because there should only be 1 header 1 on a page or we trigger accessibility failures